android-sdk/README.md

```diff
@@ -0,0 +1,168 @@
+
+# Moloco Android SDK - Android
+Moloco Android SDK provides ad serving for mobile applications to facilitate advertising for app publishers.
+
+
+## Prerequisites
+Moloco Android SDK 0.0.1 is compatible for devices running Android **API 9** and above.
```

Comment:
compatible to -> compatible with
Moloco Android SDK 0.0.1 is compatible with all Android devices running Android **API 14** and above.